onnxconverter_common/__init__.py

```diff
@@ -8,7 +8,7 @@
 This framework performs optimization for ONNX models and
 includes common utilities for ONNX converters.
 """
-__version__ = "1.6.5"
+__version__ = "1.7.0"
```

Comment:
Default opset is still 11 here when default opset in onnx is 12, version 1.7.0  or below?

Reply:
The PR mainly focus on API, and enable converter to reuse the new make_model_ex API with onnconverter-common >= 1.7.0

And default version should be updated in David's opset 12 PR @jiafatom

Reply:
Think through it, it looks we can only upgrade default_opset_version until the ORT support the new opset, otherwise, our test pipeline will be broken.